src/Jackett.Common/Indexers/SceneTime.cs

```diff
@@ -194,6 +187,10 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             var searchUrl = SearchUrl + "?" + qParams.GetQueryString();
 
             var results = await RequestStringWithCookies(searchUrl);
+            if (results.Content == null || !results.Content.Contains("/logout.php"))
+                throw new Exception("The user is not logged in. It is possible that the cookie has expired or you " +
+                                    "made a mistake when copying it. Please check the settings.");
```

Comment:
```suggestion
                throw new WebException("The user is not logged in. It is possible that the cookie has expired or you " +
                                    "made a mistake when copying it. Please check the settings.", WebExceptionStatus.ProtocolError);
```

Reply:
I don't agree. A non related exception is worse than the generic exception. If we want to fix this properly we have to create a folder/package in commons with Jackett specific exceptions. We can have meaningful exceptions for our use case like `BadLoginException`, `HtmlParsingException`...
I'm going to merge since that Exception was in old code.